scheduler/server/stateful_scheduler.go

```diff
@@ -636,6 +653,7 @@ func (s *statefulScheduler) checkJobsLoop() {
 						s.requestorHistory[rb] = []string{}
 					}
 					s.requestorHistory[rb] = append(s.requestorHistory[rb], checkJobMsg.jobDef.Tag)
+					s.requestorHistoryEntriesSize += int64(len(checkJobMsg.jobDef.Tag))
```

Comment:
is this supposed to be adding the number of entries or the length of the Tag string?